tests-ffi/include/libyrs.h

```diff
@@ -258,7 +258,7 @@ typedef Branch Branch;
 typedef union YOutputContent {
   char flag;
   float num;
-  long integer;
+  long long integer;
```

Comment:
Since `c_long` [is variable length type](https://doc.rust-lang.org/std/os/raw/type.c_long.html) (64-bit on UNIX, 32-bit on Windows), I've changed it to `c_long_long` which makes pinky promise to always be 64-bits regardless of OS.